api/src/main/java/io/opentelemetry/trace/SpanContext.java

```diff
@@ -146,14 +175,21 @@ public String toString() {
         + spanId
         + ", traceFlags="
         + traceFlags
+        + ", isRemote="
+        + isRemote
```

Comment:
just curious...why isn't this an autovalue class?

Reply:
Good question :)) Probably it should be.

Reply:
https://github.com/open-telemetry/opentelemetry-java/issues/628